joft/cli.py

```diff
@@ -16,9 +16,16 @@
 
 
 @click.group()
+@click.option("--config", help="Path to the config file.")
 @click.pass_context
-def main(ctx) -> None:
-    ctx.obj = joft.utils.load_toml_app_config()
+def main(ctx, config: str) -> None:
+    """
+    A CLI automation tool which interacts with a Jira instance and automates tasks.
+    """
+    if config:
```

Comment:
This looks redundant to me. You check `if config_path` in the `load_toml_app_config` as well so it might make sense to call it with the kw arg here no matter what's inside and handle it there?